.gitignore

```diff
@@ -2,5 +2,5 @@
 coverage
 jspm_packages
 node_modules
-public
-*.log
\ No newline at end of file
+dist
```

Comment:
- `dist/`: distribution, it usually contains the compiled software.
- `lib/`: contains the package's dependencies